### ISSUE TITLE
Provide guidance for choosing machine sizes (lol good luck)

### ISSUE THREAD

**Issue by efritz** [sourcegraph/sourcegraph#27870]

**Comment by chrismwendt** [comment]
Should this go in the [resource estimator](https://docs.sourcegraph.com/admin/deploy/resource_estimator) once executors are part of the base Sourcegraph deployment?

**Comment by macraig** [comment]
Yes, just had a conversation with @varungandhi-src about this. A first step is to have an internal spreadsheet, which we would eventually migrate to the resource estimator.
